
No static linking please. Also prevents warnings about use of dlopen().
File paths fixed for working directory.

diff --git a/gtests/net/packetdrill/Makefile.Linux b/gtests/net/packetdrill/Makefile.Linux
index 0c8b7ac..fc53fb5 100644
--- a/Makefile.Linux
+++ b/Makefile.Linux
@@ -1,2 +1,2 @@
-packetdrill-ext-libs := -lpthread -lrt -ldl -static
+packetdrill-ext-libs := -lpthread -lrt -ldl
 include Makefile.common
